documentation/asciidoc/microcontrollers/pico-series/about_pico.adoc

```diff
@@ -10,7 +10,6 @@ The original Raspberry Pi Pico family, referred to as Pico or Pico 1, comes in f
 The second-generation Raspberry Pi Pico family is referred to as Pico 2. Pico 2 comes in two variants:
 
 * Raspberry Pi Pico 2
```

Comment:
And also this little thing :stuck_out_tongue_winking_eye: 
https://www.raspberrypi.com/news/raspberry-pi-pico-2-w-on-sale-now/

Reply:
lol. nice catch. cleaned this up